src/framework/mlt_types.h

```diff
@@ -121,6 +121,21 @@ typedef enum
 }
 mlt_colorspace;
 
+typedef enum
+{
+	mlt_deinterlace_method_none,
+	mlt_deinterlace_method_bob,
```

Comment:
maybe "method" is not needed - just "mlt_deinterlace_bob" etc.

Reply:
Considering the "deinterlace_method" property is deprecated, and the new property name is "deinterlacer" these should be named `mlt_deinterlacer_` . See 312d2b3b2c44b0054a18499dc63036e0d5c1c55c and 493fa4b642099d295ea5b2b78f85a366a3d8c9ad for those changes about a year ago.

Reply:
I like that better. Done.